src/dtos/monitor-span.dto.ts

```diff
@@ -70,7 +69,7 @@ export class MonitorSpanBaseDTO {
     'SELECT distinct span_scale_code as "value" FROM camdecmpsmd.vw_span_master_data_relationships',
     {
       message: (args: ValidationArguments) => {
-        return `${args.property} The value : ${args.value} for ${args.property} is invalid`;
+        return `[${args.property}] The value : [${args.value}] for [${args.property}] is invalid`;
```

Comment:
Why does args.property appear twice in the error message?

Reply:
updated